src/connectDispatcher.ts

```diff
@@ -0,0 +1,51 @@
+import { Observable } from "rxjs"
+import shareLatest, { cache as sharedCache } from "./internal/share-latest"
+import reactEnhancer, { cache as reactCache } from "./internal/react-enhancer"
+import { useEffect } from "react"
+
+const dispatchers = new WeakMap<Observable<any>, any>()
+
+const getReactObservable$ = <T>(input$: Observable<T>) => {
+  if (!sharedCache.has(input$)) {
+    input$ = shareLatest(input$)
+  }
+  if (!reactCache.has(input$)) {
+    reactEnhancer(input$, 200)
+  }
+  return reactCache.get(input$)!
+}
```

Comment:
I'm a bit confused: Why does it need to be `shareLatest`-ed and `reactEnhancer`-ed? As I'm reading how this is being used and probably would be used by users, I don't think this should need to apply these methods:
* `shareLatest` shouldn't be needed because the `useEffect` down below will use the stream with shareLatest on it, but only only applies to that internal useEffect.
* `reactEnhancer` shouldn't be needed because it's ignoring every message, so react doesn't really play a role in the behaviour of that stream.